src/test/java/io/debezium/connector/vitess/transforms/ReplaceFieldValueTest.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.debezium.connector.vitess.transforms;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.Map;
+
+import org.apache.kafka.connect.data.Schema;
+import org.apache.kafka.connect.data.Struct;
+import org.apache.kafka.connect.source.SourceRecord;
+import org.junit.Test;
+
+import io.debezium.data.Envelope;
+import io.debezium.schema.SchemaFactory;
+
+/**
+ * @author Thomas Thornton
+ */
+public class ReplaceFieldValueTest {
+
+    private static final String TRANSACTION_BLOCK_SCHEMA_NAME = "event.block";
+    public static final int TRANSACTION_BLOCK_SCHEMA_VERSION = 1;
+
+    @Test
+    public void shouldReplaceTxIdWhenEnabled() {
+        ReplaceFieldValue<SourceRecord> removeFields = new ReplaceFieldValue<SourceRecord>();
```

Comment:
`removeFields` or `replaceFields`? The same issue is present in other methods too.